Keas.Mvc/Controllers/Api/EquipmentController.cs

```diff
@@ -63,6 +63,38 @@ select new
             return Json(await equipment.ToListAsync());
         }
 
+        [HttpGet]
+        [ProducesResponseType(typeof(IEnumerable<Equipment>), StatusCodes.Status200OK)]
+        public async Task<IActionResult> SearchAttributes(string q)
+        {
+            var attributes = from ea in _context.EquipmentAttributes
+                    .Where(a => a.Equipment.Team.Slug == Team && a.Equipment.Active &&
+                                (EF.Functions.Like(a.Value, q.EfStartsWith())))
+                    .Include(a => a.Equipment)

```

Comment:
.Include doesn't do anything if you are going to do a `select new { id }` anyway